app/code/Magento/Checkout/ViewModel/Cart.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Checkout\ViewModel;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\View\Element\Block\ArgumentInterface;
+use Magento\Framework\View\Element\Context;
+use Magento\Store\Model\ScopeInterface;
+
+class Cart implements ArgumentInterface
+{
+    /**
+     * Config settings path to enable clear shopping cart button
+     */
+    public const XPATH_CONFIG_ENABLE_CLEAR_SHOPPING_CART = 'checkout/cart/enable_clear_shopping_cart';
```

Comment:
We try to avoid public, protected variable or function. Please change this public constant to private.

Reply:
Updated to private const